### PR TITLE
feat: add author page and new profile fields

### PR DESCRIPTION
**Pull request by abepeterkin**
New profile fields:
Screen Shot 2020-12-16 at 10 26 17 PM

Desktop page:
Screen Shot 2020-12-16 at 10 26 59 PM

Mobile page:
Screen Shot 2020-12-16 at 10 28 14 PM

### CODE REVIEW

**Review by hellatan — APPROVED** [review]

File: due-processed/author.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+			<!-- bio -->
+			<div class='author-profile-bio'>
+				<?php echo $curauth->user_description; ?>
+			</div>
+
+			<!-- contact information -->
+			<div class='author-profile-body-right'>
+				<?php if ( get_the_author_meta( 'twitter' )) { ?>
+					<div class='author-profile-contact-info'>
+						<img class='author-profile-icon' src='<?= get_template_directory_uri() ?>/icons/twitter.svg'/>
```
Comment (lilianammmatos) [inline due-processed/author.php:58]: What do we think about putting SVGs in PHP templates instead ([example](https://github.com/DueProcessed/dueprocessed.com/pull/54/files#diff-993fdc340f41e0b0da97e7c67477a174f3554f0d3edd48ce345a74d9171cf22dR35))? I tried the `img` approach here, but it didn't allow me to change the color of the icons with CSS (which I need to do for the search icon).

Comment (abepeterkin) [reply due-processed/author.php]: Updated to use PHP templates

File: due-processed/template-parts/icon-twitter.php [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg class="icon" width="18" height="15" viewBox="0 0 18 15" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (lilianammmatos) [inline due-processed/template-parts/icon-twitter.php:1]: remove extra attributes
```suggestion
<svg class="icon" viewBox="0 0 18 15">
```

File: due-processed/template-parts/icon-twitter.php [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
...2.65198 12.975 4.22026 12.4816 5.58003 11.5066C4.00294 11.4714 2.66667 10.3612 2.04993 8.82819C2.22026 8.92511 2.45228 8.98091 2.73128 8.98091C2.84581 8.98091 3.03671 8.95154 3.28634 8.90162C3.52717 8.85756 3.6975 8.82819 3.79736 8.82819C2.03231 8.53157 0.713656 7.04552 0.713656 5.21586C1.22467 5.42144 1.36858 5.49192 1.79736 5.57122C2.01175 5.61527 2.23789 5.63289 2.4699 5.63289C1.43906 4.98678 0.713656 3.81498 0.713656 2.65786C0.713656 1.77092 0.866373 1.24229 1.29809 0.640235C3.08076 2.94567 5.83847 4.39648 8.78708 4.60206C8.78708 4.48752 8.77533 4.33186 8.76946 4.14391C8.75771 3.95595 8.75183 3.82085 8.75183 3.72394C8.75183 2.38767 9.4978 1.15418 10.6167 0.490455C11.1806 0.161527 11.7709 0 12.4023 0C13.4244 0 14.373 0.396476 15.2511 1.19236C15.7504 1.13069 16.2203 0.951542 16.8722 0.654919C17.1953 0.502203 17.4273 0.402349 17.5624 0.349486C17.2863 1.14537 16.7372 1.721 15.7974 2.37298C16.6667 2.28194 17.4009 2.07636 18 1.76505C17.6153 2.43759 17.0426 3.08957 16.279 3.72687V4.1674C16.279 5.77974 15.8326 7.46549 15.0338 9.03084C14.2379 10.5962 12.9457 11.9971 11.4332 13.0279C9.9207 14.0587 7.88546 14.6843 5.70631 14.6843C3.63583 14.6843 1.72981 14.1116 0 12.975Z" fill="#151515"/>
```
Comment (lilianammmatos) [inline due-processed/template-parts/icon-twitter.php:2]: remove `fill` at the end
```suggestion
	<path d="M0 12.975H0.87812C2.65198 12.975 4.22026 12.4816 5.58003 11.5066C4.00294 11.4714 2.66667 10.3612 2.04993 8.82819C2.22026 8.92511 2.45228 8.98091 2.73128 8.98091C2.84581 8.98091 3.03671 8.95154 3.28634 8.90162C3.52717 8.85756 3.6975 8.82819 3.79736 8.82819C2.03231 8.53157 0.713656 7.04552 0.713656 5.21586C1.22467 5.42144 1.36858 5.49192 1.79736 5.57122C2.01175 5.61527 2.23789 5.63289 2.4699 5.63289C1.43906 4.98678 0.713656 3.81498 0.713656 2.65786C0.713656 1.77092 0.866373 1.24229 1.29809 0.640235C3.08076 2.94567 5.83847 4.39648 8.78708 4.60206C8.78708 4.48752 8.77533 4.33186 8.76946 4.14391C8.75771 3.95595 8.75183 3.82085 8.75183 3.72394C8.75183 2.38767 9.4978 1.15418 10.6167 0.490455C11.1806 0.161527 11.7709 0 12.4023 0C13.4244 0 14.373 0.396476 15.2511 1.19236C15.7504 1.13069 16.2203 0.951542 16.8722 0.654919C17.1953 0.502203 17.4273 0.402349 17.5624 0.349486C17.2863 1.14537 16.7372 1.721 15.7974 2.37298C16.6667 2.28194 17.4009 2.07636 18 1.76505C17.6153 2.43759 17.0426 3.08957 16.279 3.72687V4.1674C16.279 5.77974 15.8326 7.46549 15.0338 9.03084C14.2379 10.5962 12.9457 11.9971 11.4332 13.0279C9.9207 14.0587 7.88546 14.6843 5.70631 14.6843C3.63583 14.6843 1.72981 14.1116 0 12.975Z"/>
```

File: due-processed/template-parts/icon-email.php [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<svg class="icon" width="18" height="12" viewBox="0 0 18 12" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (lilianammmatos) [inline due-processed/template-parts/icon-email.php:1]: remove extra attributes
```suggestion
<svg class="icon" viewBox="0 0 18 12" >
```

File: due-processed/template-parts/icon-email.php [hunk 5]
```diff
@@ -0,0 +1,3 @@
...
+	<path d="M17.9927 0.326416L-0.00732422 0.355216L0.00707578 11.6736L18.0071 11.6448L17.9927 0.326416ZM16.6463 0.981616L8.99988 7.27442L1.33908 1.00322L16.6463 0.981616ZM0.655076 11.0256L0.640676 1.26962L8.99988 8.10962L17.3447 1.24082L17.3591 10.9968L0.655076 11.0256Z" fill="black"/>
```
Comment (lilianammmatos) [inline due-processed/template-parts/icon-email.php:2]: remove `fill` at end
```suggestion
	<path d="M17.9927 0.326416L-0.00732422 0.355216L0.00707578 11.6736L18.0071 11.6448L17.9927 0.326416ZM16.6463 0.981616L8.99988 7.27442L1.33908 1.00322L16.6463 0.981616ZM0.655076 11.0256L0.640676 1.26962L8.99988 8.10962L17.3447 1.24082L17.3591 10.9968L0.655076 11.0256Z"/>
```

**Review by lilianammmatos — APPROVED** [review]
small clean-up comments for SVGs, but otherwise LGTM 👍

**Pull request merged**